docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/ZAP.xml

```diff
@@ -58,6 +58,63 @@
     handling your support request, or you can upload the file into the issue
     tracking system. Remember, you should only use the email feature if
     appropriate to share your data.</para>
+
+    <sect3 id="ZAP_Config_Options">
+      <title>Configuration Options</title>
+
+      <para>The component logs are some of the most important artifacts
+      included in these ZAP reports. </para>
+
+      <para>The logging behavior differs on bare-metal and containerized
+      deployments. In bare metal, Some logs are always slated to be included
+      (Thormaster, ECLCC Server, and ECL Agent). The UI checkbox “Include
+      worker logs” controls the inclusion of the Thor worker (fka Thor slave)
+      log files. </para>
+
+      <para>In a containerized environment, you can exclude all logs by
+      de-selecting the UI check boxes. Containerized deployments allow this
+      flexibility because logs require a remote log reader component which may
+      not be configured on your system. Collection and downloading of logs may
+      incur costs you wouldn’t incur on bare metal, one reason why logging
+      might not be enabled on some cloud deployments. </para>
```

Comment:
The last sentence should be reworded. Here is a suggestion:
Cloud-based logging can lead to additional expenses for data collection and downloads compared to traditional servers, which is why some cloud deployments may have logging disabled.